### ISSUE TITLE
libwasmvm.so is missing for non-Docker setup

### ISSUE THREAD

**Issue by vorotech** [osmosis-labs/osmosis#1284]
### What is the issue?

`libwasmvm.so` is missing when executing `osmosisd` binary downloaded from the releases page.

This is not a Docker based, but Full Node setup

```sh
$ osmosisd init moniker-1 --home=/home/cosmos/.osmosisd --chain-id=osmosis-1
...
osmosisd: error while loading shared libraries: libwasmvm.so: cannot open shared object file: No such file or directory
```

### System

```sh
$ uanme -a
Linux #23~20.04.2-Ubuntu SMP Thu Mar 31 11:36:15 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
$ go version
go version go1.18.1 linux/amd64
```

`osmosisd` is https://github.com/osmosis-labs/osmosis/releases/download/v7.2.0/osmosisd-7.2.0-linux-amd64

Last known working version for both downloaded binary as well as built from source without any issues is v6.4.0

### Workaround 

The only workaround I've found is to build from sources, but make extra steps similar to those in Docker file

```sh
wget https://github.com/CosmWasm/wasmvm/releases/download/v1.0.0-beta7/libwasmvm_muslc.a -O /lib/libwasmvm_muslc.a
git clone https://github.com/osmosis-labs/osmosis
cd osmosis
BUILD_TAGS=muslc make build
cp ./build/osmosisd /usr/local/bin
```

NOTE: Only version `v1.0.0-beta7` of CosmWasm/wasmvm is working, other (newer) has breaking interface changes

### Expected result

The `osmosisd` downloaded from the releases page works without any additional requirements or at lease this process is documented.

**Comment by p0mvn** [comment]
Thank you for this issue. It seems that our [builder image](https://github.com/osmosis-labs/osmosis/blob/298ad00dc9f792dfccf10e864002a02bcdd0b3fb/contrib/images/rbuilder/Dockerfile) was not updated with the  cosm wasm related dependencies. I will look into it

**Comment by alexanderbez** [comment]
Does this have anything to do with cross-compiling? If so, enabling CGO and shared libs in automated builds that cross compile is not possible AFAIK (e.g. this is why Ledger doesn't work on cross-compiled builds).

**Comment by ValarDragon** [comment]
My understanding is that cosmwasm statically links the wasmvm binary on supported systems, but on any unsupported system it wouldn't, and requires it to be dynamically linked.
Likely the system this release was made for then didn't have a static link available, and so assumed it would be dynamically linked. (Which it wasn't - hence the failure)

**Comment by p0mvn** [comment]
> My understanding is that cosmwasm statically links the wasmvm binary on supported systems, but on any unsupported system it wouldn't, and requires it to be dynamically linked. Likely the system this release was made for then didn't have a static link available, and so assumed it would be dynamically linked. (Which it wasn't - hence the failure)

That's correct. However, our build scripts were broken and didn't statically link the binary on the supported system (`linux/amd64`). It is now fixed in #1286 

I updated the `v7.2.0` release binary and tested it locally with no issues. @vorotech please let us know if you have issues with the newly uploaded binaries

**Comment by vorotech** [comment]
@p0mvn I've tested the new binary of `v7.2.0` and it works fine now 👍
